### DISCUSSION TITLE
关于armbian系统安装OMV（NAS系统）教程

### DISCUSSION THREAD

**Discussion by Yxini** [ophub/amlogic-s9xxx-armbian#1799] [Other]
此教程参考于[Abin17618](https://blog.csdn.net/Abin17618)的R3300L安装armbian（bullseye [23.08.0）并装OMV](https://blog.csdn.net/Abin17618/article/details/131738167)，已发私信给Abin17618，如有侵犯会立即删除。
1、首先安装好armbian系统，然后更换更换国内源：
我使用的是LinuxMirrors的一键更换执行命令[https://supermanito.github.io/LinuxMirrors/use/](url)
执行此命令：
`bash <(curl -sSL https://linuxmirrors.cn/main.sh)`
2、更新软件：
执行此命令：
`apt update && apt upgrade -y`
3、手动安装 openmediavault 密钥环：
执行以下命令：
```
apt-get install --yes gnupg
wget -O "/etc/apt/trusted.gpg.d/openmediavault-archive-keyring.asc" https://packages.openmediavault.org/public/archive.key
apt-key add "/etc/apt/trusted.gpg.d/openmediavault-archive-keyring.asc"
```
4、添加清华源软件包储存库加速
部分参考了[https://mirrors.tuna.tsinghua.edu.cn/help/openmediavault/](url)
执行以下命令：
```
cat <<EOF > /etc/apt/sources.list.d/openmediavault.list
deb https://mirrors.tuna.tsinghua.edu.cn/OpenMediaVault/public shaitan main
deb https://mirrors.tuna.tsinghua.edu.cn/OpenMediaVault/packages shaitan main
## Uncomment the following line to add software from the proposed repository.
# deb https://mirrors.tuna.tsinghua.edu.cn/OpenMediaVault/public shaitan-proposed main
# deb https://mirrors.tuna.tsinghua.edu.cn/OpenMediaVault/packages shaitan-proposed main
## This software is not part of OpenMediaVault, but is offered by third-party
## developers as a service to OpenMediaVault users.
# deb https://mirrors.tuna.tsinghua.edu.cn/OpenMediaVault/public shaitan partner
# deb https://mirrors.tuna.tsinghua.edu.cn/OpenMediaVault/packages shaitan partner
EOF
```
5、启动科学环境或者使用Windows启动clash，使用clash的http代理端口
可文件里面最下面添加下面三条命令：（其中192.168.2.240更换为Windows的IP）
```
nano /etc/profile
export http_proxy="http://192.168.2.240:7890"
export https_proxy="http://192.168.2.240:7890"
export ftp_proxy=$http_proxy
```
6、OMV自动安装的脚本，装完后包含omv-extras插件：
执行以下命令：
`wget -O - https://github.com/OpenMediaVault-Plugin-Developers/installScript/raw/master/install | sudo bash`
可能要等不少于半小时吧，根据视网络情况而定。搞定后会自动重启。
看到这条就说明安装完毕了等重启即可
`IP address may change and you could lose connection if running this script via ssh  `                                                                                                                        
PS：安装到最后可能会更换IP，这个时候进路由器看一下新的IP地址即可。
![图片](https://github.com/ophub/amlogic-s9xxx-armbian/assets/47093205/91371570-34a5-4184-a2d6-ff9b8bd57575)

在浏览器输入http://机顶盒的ip地址就可以打开OMV的登陆界面了，登录的默认账号是admin，默认密码是openmediavault。
到此就安装完毕了。

**Comment by ophub** [comment]
感谢分享经验
